### ISSUE TITLE
Add additional modes 

### ISSUE THREAD

**Issue by beagleboys** [0x4a616e/homebridge-smartthings-ac#5]
Hi, first of all: great work! your implementation is way faster than event the official iOS app!

Do you think is possibile to add the missing modes(Fan, Dry) and maybe separate buttons for the wind free and air purification modes?

For what regards the missing modes they are getting trough to home bridge, I have this errors showing:
`[HomebridgeSmartThingsAC] Received unknown heater-cooler state wind`
`[HomebridgeSmartThingsAC] Received unknown heater-cooler state dry`

**Comment by 0x4a616e** [comment]
Yeah, we could probably add an additional fan and air dehumidifier accessory in HomeKit for each AC. Than, the fan accessory would e.g. be on if the "physical" AC is on and in mode `fan`, all other devices would be shown as off. The good thing in this approach is that we could "officially" support air humidity which is not supported for ACs.

The other option would be additional switches. The problem is, what mode should be shown for the AC? There are just the basic options...

**Comment by beagleboys** [comment]
the thermostat modes limited to the auto, cool and heat modes then? or you can add modes there?
<img width="306" alt="Schermata 2021-08-30 alle 18 39 32" src="https://user-images.githubusercontent.com/1041560/131373869-93cd9865-d76f-4003-a5c9-bc449b8fb235.png">

**Comment by 0x4a616e** [comment]
Yes, unfortunately those are the only modes supported by HomeKit for air conditioner accessories https://developers.homebridge.io/#/characteristic/TargetHeaterCoolerState
